core-addon/Core.js

```diff
@@ -7,6 +7,10 @@ const DataCollection = require("./DataCollection.js");
 
 // The path of the embedded resource used to control options.
 const OPTIONS_PAGE_PATH = "public/index.html";
+// The website to post deletion IDs to.
+const OFFBOARD_URL = "https://production.rally.mozilla.org/offboard";
+// The static website to send offboarded users to (those with no deletion ID).
+const LEAVE_URL = "https://rally.mozilla.org/leaving-rally/";
```

Comment:
```suggestion
const LEAVE_URL = "__BASE_SITE__/leaving-rally";
```
I think this can probably remain based on the provided base site.